content/sensu-go/6.2/release-notes.md

```diff
@@ -74,6 +75,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.2.0 release notes
+
+**December 17, 2020** &mdash; The latest release of Sensu Go, version 6.2.0, is now available for download.
+
+**ADD RELEASE SYNOPSIS HERE**
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.2.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][193]) Added support for the `memberof` attribute for the [LDAP authentication provider][199].
+- ([Commercial feature][193]) Added the ability to exclude resource types when using sensuctl prune with the [--omit flag][200].
+- ([Commercial feature][193]) Added support for [round robin scheduling on PostgreSQL][201] instead of etcd.
+- ([Commercial feature][193]) Added support for OIDC authentication via [sensuctl configure][202].
+- The [/metrics API endpoint][196] now exposes build information as a Prometheus metric.
+- Added /health endpoint to agentd.
+- Checks now include the [`scheduler` attribute][197], which Sensu automatically populates with the type of scheduler that schedules the check.
+- Events now include the [`sequence` attribute][198], which the Sensu agent automatically sets at startup and increments by 1 at every successive check execution or keepalive event.
+- Added support for using environment variables to define the configuration file paths for the Sensu agent (`SENSU_CONFIG_FILE`) and backend (`SENSU_BACKEND_CONFIG_FILE`).
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][193]) Refactored entity limiter to ensure that warning messages about approaching a license's entity or entity class limit are now only displayed for users with `create` or `update` permissions for the license.
+- Adjusted the format for silenced entry dates and durations in sensuctl. For all silenced entries, the begin date is now listed in RFC 3339 format. For silenced entries that have not begun, the list displays the expiration date in RFC 3339 format. For silenced entires with no expiration date, the list displays `-1`. For silenced entries that have begun, the list displays the duration (e.g. 1m30s).
```

Comment:
```suggestion
- Adjusted the format for silenced entry dates and durations in sensuctl tabular formatted output. For all silenced entries, the begin date is now listed in RFC 3339 format. For silenced entries that have not begun, the list displays the expiration date in RFC 3339 format. For silenced entires with no expiration date, the list displays `-1`. For silenced entries that have begun, the list displays the duration (e.g. 1m30s).
```

Reply:
Ideally we would link to an explanation of RFC 3339, but the most obvious route of linking to Wikipedia winds up at https://en.wikipedia.org/wiki/ISO_8601 which might be even more confusing. 🤔

Reply:
Yeah. There is also the [golang time page](https://golang.org/pkg/time/#pkg-constants), but there's no good heading for linking straight to RFC3339 (readers would have to dig)